CONTRIBUTING.rst

```diff
@@ -57,20 +52,20 @@ If you are proposing a feature:
 Get Started!
 ------------
 
-Ready to contribute? Here's how to set up `xhydro` for local development.
+Ready to contribute? Here's how to set up ``xhydro`` for local development.
 
-1. Fork the `xhydro` repo on GitHub.
-2. Clone your fork locally::
+#. Fork the ``xhydro`` repo on GitHub.
+#. Clone your fork locally::
 
     $ git clone git@github.com:your_name_here/xhydro.git
```

Comment:
`xscen` only asks people to clone, while here we want them to fork first. Is this right?

Reply:
This is the cookiecutter default, but _from my understanding,_ there's a higher likelihood that contributors to this project might be coming from outside `Hydrologie`. For one-off contributors, this is what we would expect.

Reply:
Right. In that case, since we've already whitelisted many expected contributors, could we add something like:
`If you are not already an xhydro collaborator, fork...`